shared/src/state-migrations/16-add-meta-position.ts

```diff
@@ -0,0 +1,106 @@
+import type { UUID } from '../utils';
+import type { Migration } from './migration-functions';
+
+export const addMetaPosition16: Migration = {
+    actions: null,
+    state: (state) => {
+        const typedState = state as {
+            patients: {
+                [patientId: UUID]: {
+                    position?: { x: number; y: number };
+                    vehicleId?: UUID;
+                    metaPosition?: any;
+                };
+            };
+            materials: {
+                [materialId: UUID]: {
+                    position?: { x: number; y: number };
+                    vehicleId?: UUID;
+                    metaPosition?: any;
+                };
+            };
+            vehicles: {
+                [vehicleId: UUID]: {
+                    position?: { x: number; y: number };
+                    transfer?: any;
+                    metaPosition?: any;
+                };
+            };
+            personnel: {
+                [personnelId: UUID]: {
+                    position?: { x: number; y: number };
+                    transfer?: any;
+                    vehicleId?: UUID;
+                    metaPosition?: any;
+                };
+            };
+        };
+
+        Object.values(typedState.patients).forEach((patient) => {
+            if (patient.position) {
+                patient.metaPosition = {
+                    type: 'coordinates',
+                    position: { x: patient.position.x, y: patient.position.y },
+                };
+            } else if (patient.vehicleId) {
+                patient.metaPosition = {
+                    type: 'vehicle',
+                    vehicleId: patient.vehicleId,
+                };
+            }
+        });
```

Comment:
I think these conditions are not exhaustive. When importing [this scenario](https://github.com/hpi-sam/digital-fuesim-manv_test-scenarios/blob/main/exercise-state-2022-05-31-workshop-babz-gro%C3%9Fes-szenario.json), there are patients that do not have a `metaPosition` after the migrations.

Reply:
It seems like the migrations are migrating the initial state and then re-apply all actions on this initial state. So you have to migrate the elements in all add-actions to use `metaPosition` in addition to `position`.

Reply:
Fixed

Reply:
This was not due to those lines, but due to missing but due to missing Action Migrations